model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -94,6 +95,22 @@ class LicenseInfoResolver(
                 originalDeclaredLicenses.addAll(
                     licenseInfo.declaredLicenseInfo.processed.mapped.filterValues { it == license }.keys
                 )
+
+                licenseInfo.declaredLicenseInfo.authors.takeIf { it.isNotEmpty() }?.let {
+                    locations.add(ResolvedLicenseLocation(
+                        provenance = Provenance(),
+                        location = UNDEFINED_TEXT_LOCATION,
+                        appliedCuration = null,
+                        matchingPathExcludes = emptyList(),
+                        copyrights = licenseInfo.declaredLicenseInfo.authors.mapTo(mutableSetOf()) {
+                            ResolvedCopyrightFinding(
+                                statement = "$COPYRIGHT_PREFIX $it",
+                                location = UNDEFINED_TEXT_LOCATION,
+                                matchingPathExcludes = emptyList()
+                            )
+                        }
+                    ))
+                }
```

Comment:
This introduces "artifical" resolved license locations which in turn might introduce some "magic" into the code consuming the instance as soon as they want to handle the authors a bit different. What I could imagine e.g. is that one would want the NOTICE template to list the author copyrights separate from the licenses. This is why I believe that 

(1) the license info resolver should have a function to get the authors for a package.
(2) the `FreemarkerTemplateProcessor.PackageModel` should be extended by a property for the authors
(3) the notice template should be extended to iterate over the authors and insert them as needed, and thus
(4) The "Copyright (C)" should also go into the template, or at least into the reporter.

Reply:
@mnonnenmacher @sschuberth any opinions on the design considerations above?

Reply:
Aren't copyrights also visible in the web app report? Or isn't that at least planned? Anyway, independently of where copyrights are shown *right now*, I agree that this should be solved at the data model level, not at the reporter level, to be independent of the report format and any further processing. Such further processing could e.g. be the discussed `copyrightFindingCurations` that analogously to `licenseFindingCurations` could be added to `PackageConfiguration` at some point.

TL;DR while I agree that the `UNDEFINED_TEXT_LOCATION` is not nice, this is the best we can do for now without significant changes to the data model. I'm fine with the changes.

Reply:
> I proposed this to actually solve the problem via the data model

Well, by the name, yes. But it's the `FreemarkerTemplateProcessor`-specific data model, not the "real" / global ORT data model.

> without mixing copyright findings with authors.

At least in my view the goal always was to *do* turn authors into (some kind of) copyright findings; these copyrights were just not found by the scanner, but the analyzer.

It's just a bit unfortunate that we do not formally distinguish between declared and detected copyrights, like we do for licenses, and have an intermediate step that turn authors into declared copyrights.

Reply:
Sorry. only following the discussion half as I am not familiar with this bit of Kotlin code/Ort mofel but it looks to me that we artificially including authors in detected findings by giving it a fake text location and prefix it with "Copyright". I believe this not the right way to do this - prefer @fviernau to leave it up to reporter to choose how to display authors.

In the WebApp I was thinking to merger `Details` and `Licenses` sections into just `Detials` and the `authors` under the  a `Declared` sub-section

Reply:
As agreed in the developer meeting I've created an [issue](https://github.com/oss-review-toolkit/ort/issues/3780) to improve this later.